test/types/enum/ferguson/enum-overflow.compopts

```diff
@@ -0,0 +1 @@
+--ignore-errors-for-pass
```

Comment:
Similar here - I'd rather take out the writeint calls and just replace them with writelns.